README.md

```diff
@@ -1 +1,51 @@
-# it_is_wrong
\ No newline at end of file
+<div align="center">
+  <img alt="DSC-SIST Logo" src="https://i.imgur.com/BjAjNjo.png" height="200" />
+</div>
+<br>
+<p align="center">
+Find the Bugs. Solve the Code ✔✨
+</p>
+<p align="center">
+Developed with <span style="color: #8b0000;">&hearts;</span> by your friends at <a href="https://srmkzilla.net">Developer Student Club- Sathyabama IST</a> Team.
```

Comment:
Use this link for hyptertext reference:

https://dsc.community.dev/sathyabama-institute-of-science-technology/